### PR TITLE
fix 5195

### LINKED ISSUE #5195: Meeting issue - "You're not authorized to see this video"
url: https://staging.archipelmarket.com/c/o/marketplace/event/5BHhDZJQYHblOJlly84N/session
users: dev+mabel-imk@blockframes.io (organizer) / dev+raymond-ndc@blockframes.io (invitee)
sentry error: https://sentry.io/organizations/cascade-8/issues/2266940312/?project=1540126&referrer=slack
video: "test-video.mp4" in the middle

Image was shared fine but then the video keeps spinning on the invitee side while playing nicely for the organizer. Sentry error "You're not authorized to see this video" spawned and from there I couldn't see any file as the invitee.

Other related errors:
https://sentry.io/organizations/cascade-8/issues/2266756083/?project=1540126&referrer=slack
https://sentry.io/organizations/cascade-8/issues/2266751082/?project=1540126&referrer=slack

### PR DESCRIPTION
**Pull request by pldespaigne**
fix #5195

### CODE REVIEW

File: apps/backend-functions/src/internals/media.ts [hunk 1]
```diff
@@ -13,9 +13,12 @@ import { StorageFile } from '@blockframes/media/+state/media.firestore';
 import { db } from './firebase';
 import { isUserInvitedToEvent } from './invitations/events';
 import { MovieDocument } from '../data/types';
+import { Privacy } from '@blockframes/utils/file-sanitizer';
 
 export async function isAllowedToAccessMedia(file: StorageFile, uid: string, eventId?: string): Promise<boolean> {
 
+  console.log('IS ALLOWED ?', JSON.stringify(file), uid, '**' + eventId + '**');
```
Comment (RemcoSimonides) [inline apps/backend-functions/src/internals/media.ts]: 🐱‍🏍

Comment (phpgeekfr) [inline apps/backend-functions/src/internals/media.ts:20]: needed ?  you can also use firestore internal logging system

**Review by phpgeekfr — APPROVED** [review]

File: apps/backend-functions/src/internals/media.ts (hunk 1 above)
Comment (pldespaigne) [reply apps/backend-functions/src/internals/media.ts:20]: good catch ! ^^

File: apps/backend-functions/src/internals/media.ts [hunk 2]
```diff
@@ -28,9 +31,26 @@ export async function isAllowedToAccessMedia(file: StorageFile, uid: string, eve
   // We should not trust `privacy` & `storagePath` that comes from the parameters
   // instead we use `collection`, `docId` & `field` to retrieve the trusted values form the db
   const docData = await getDocument(`${file.collection}/${file.docId}`);
-  const storageFile: StorageFile | undefined = get(docData, file.field);
-  if (!storageFile) { return false; }
-  const { privacy, storagePath } = storageFile;
+  const storageFile: StorageFile | StorageFile[] | undefined = get(docData, file.field);
+
+  let privacy: Privacy = 'protected';
+  let storagePath = 'unknown';
```
Comment (RemcoSimonides) [inline apps/backend-functions/src/internals/media.ts:35]: No need to set it to `unknown` I guess?

**Review by RemcoSimonides — APPROVED** [review]

**Pull request merged**